cve_bin_tool/config.py

```diff
@@ -3,28 +3,47 @@
 
 from __future__ import annotations
 
+import sys
 from collections import ChainMap
 from logging import Logger
 from pathlib import Path
 from typing import Mapping
 
-import toml
+if sys.version_info >= (3, 11):
+    import tomllib as toml
+else:
+    import toml
+
 import yaml
 
 from cve_bin_tool.error_handler import ErrorHandler, ErrorMode, UnknownConfigType
 from cve_bin_tool.log import LOGGER
 
 
 class ConfigParser:
-    # Key-value pair of config data ex: {"extract": True, "directory": "test/assets"}
+    """
+    Parses configuration files in either TOML or YAML format.
+    """
+
     config_data: Mapping[str, dict[str, dict[str, str | bool | int | list[str]]]]
+    """
+    Key-value pairs of config data, e.g., {"extract": True, "directory": "test/assets"}.
+    """
```

Comment:
```suggestion
    # Key-value pairs of config data, e.g., {"extract": True, "directory": "test/assets"}.
```

This should be a comment rather than a docstring.  It's a useful comment, though!  I just don't think there's any convention for docstrings after variables the way there is after function/class/module definitions.  (Feel free to double check https://peps.python.org/pep-0257/ and let me know if my knowledge is outdated, though.  Sometimes I miss a new pep!)